client/my-sites/email/email-management/home-page.tsx

```diff
@@ -1,9 +1,11 @@
 import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
 import EmailHome from 'calypso/my-sites/email/email-management/email-home';
+import { IntervalLength } from 'calypso/my-sites/email/email-providers-comparison/interval-length';
 import { emailManagement } from 'calypso/my-sites/email/paths';
 
 type EmailManagementHomePageProps = {
 	selectedDomainName: string;
+	selectedIntervalLength: IntervalLength;
```

Comment:
Don't we also need to handle `undefined` here?
```suggestion
	selectedIntervalLength?: IntervalLength;
```